app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -429,6 +486,15 @@ public void openSentSurvey() {
         initSurvey();
     }
 
+    public void hideReview() {
+        isMoveToLeft=true;
+        restoreAssess();
+    }
+
+    public boolean isLoadingReview() {
```

Comment:
@idelcano why do we need a getter if isLoadingReview variable is public? Either make the variable private (recomended) or keep it public and remove the getter....